src/components/ContactsList/ContactsListItem.vue

```diff
@@ -156,13 +174,20 @@ export default {
 				params: { selectedGroup: this.selectedGroup, selectedContact: this.source.key },
 			})
 		},
+		onSelectMultiple() {
+			// This weirdness of passing a function as a prop is because the VirtualList extra-props prop object does not support listening to custom events (afaik)
```

Comment:
Why not use an event bus ?

Reply:
I think it's discouraged in Vue3, having said that what I'm doing here isn't exactly standard either, but it might be more future proof.